docs/src/content/blog/the-fresh-keystatic-cli.mdoc

```diff
@@ -0,0 +1,24 @@
+---
+title: The fresh Keystatic CLI
+draft: false
+publishedOn: '2023-06-28'
+summary: >-
+  Kick-start a Keystatic-integrated project with our new "create" CLI, available
+  via NPM.
+authors:
+  - simon-swiss
+  - johan-brolin
+---
+The freshly baked Keystatic CLI is now published on NPM 🔥
+
+Simply run `npm create @keystatic@latest` in your terminal to try it out!
```

Comment:
```suggestion
You can run `npm create @keystatic@latest` in your terminal to try it out!
```

Reply:
I don't mind it myself, but I've heard many time that one should avoid words like "just", "simply" and things that assume everyone is fluent with terminal stuff.

Reply:
Great point – thanks @simonswiss 🙌🏻